focuspoints.lrdevplugin/PentaxDelegates.lua

```diff
@@ -151,6 +153,8 @@ function PentaxDelegates.getAfPointsContrast(photo, metaData)
         })
       end
     end
+  elseif (contrastAfMode[1] == "Face Detect AF" and facesDetected == 0) then
+    LrErrors.throwUserError("Face Detect AF mode enabled, but no faces detected.")
```

Comment:
Is it appropriate to throw an error here? The plugin can't show any focus points in this state?